src/typings/index.d.ts

```diff
@@ -0,0 +1,39 @@
+type PokemonCard = {
+  id: string
+  name: string
+  supertype: string
+  types: string[]
+  images: {
+    small: string
+    large: string
+  }
+  set: {
+    releaseDate: string
+  }
+  flavorText: string
+  tcgplayer?: {
+    url: string
+  }
+}
+
+type ListCardsResponse = {
+  data: PokemonCardObject[]
+  page: number
+  pageSize: number
+  count: number
+  totalCount: number
+}
+
+type Collection = {
+  cards: PokemonCard[]
+}
+
+type isCollectedArgs = {
```

Comment:
always put arg/prop types inline above there functions. You don't want this in the global namespace